src/plugins/intel_gna/tests/functional/shared_tests_instances/subgraph_tests/permute_concat_permute.cpp

```diff
@@ -10,9 +10,12 @@ using namespace SubgraphTestsDefinitions;
 
 namespace {
 std::vector<std::vector<std::vector<size_t>>> inputs{
+    {{1, 8}, {1, 0}, {1, 0}},
+};
+
+std::vector<std::vector<std::vector<size_t>>> inputsNeg{
     {{32, 2}, {1, 0}, {1, 0}},
     {{8, 2}, {1, 0}, {1, 0}},
-    {{1, 8}, {1, 0}, {1, 0}},
```

Comment:
Why are some tests moved from negative?

Reply:
Because the problem does not apply to the trivial cases so there is no throw from the plugin.